nebula-logger/main/log-management/classes/LogEntryEventHandler.cls

```diff
@@ -36,13 +37,13 @@ public without sharing class LogEntryEventHandler {
      * @description Runs the trigger handler's logic for the `LogEntryEvent__e` platform event object
      */
     public void execute() {
-        switch on this.triggerOperationType {
-            when AFTER_INSERT {
-                this.upsertLogs();
-                this.insertLogEntries();
-                this.insertTopics();
-            }
+        if (this.logEntryEvents.isEmpty()) {
```

Comment:
something that I think helps with the readability/avoidance of an early return is inverting the conditional - if they list is _not_ empty, upsert logs, insert log entries, etc ...

Reply:
You've convinced me! I've flipped it around so it calls the private methods if the list is not empty.